returnn/datasets/distrib_files.py

```diff
@@ -308,62 +351,62 @@ def _get_sub_dataset_dict(self, files: List[FileTree]) -> Tuple[Dict[str, Any],
         return dataset_dict, exit_hook
 
     @staticmethod
-    def _get_files_per_sub_epochs(
-        *, partition_epoch: int, file_sizes: Dict[str, int], files_order: Sequence[FileTree]
+    def _distribute_evenly_by_size(
+        *, num_bins: int, file_sizes: Dict[str, int], files_order: Sequence[FileTree]
     ) -> List[List[FileTree]]:
```

Comment:
If possible try to add a docstring that at least specifies that the `num_bins` should be something similar to either the `partition_epoch` parameter, or the number of workers, or whatever the user specifies.